Initial draft of design team goals

quinnkeast opened the pull request:
Based on our design team offsite, we've identified an [initial set](https://www.figma.com/file/CKDSU7XGD0r5QljyoRCHOg/Lisbon-Design-Offsite-Workshops?node-id=116%3A729&t=vTqCx0evZNP0CCIn-4) of "urgent and important" and "important but non-urgent" goals for the design team over the next 3-6 months, cascading down from the business vision and objectives.

This is the initial draft of these goals for the handbook. I've directly copied over the rough work we did in Figjam. We'll need to refine these further from here.

quinnkeast committed:
Initial draft of design team goals

quinnkeast committed:
squash! Prettier

jjinnii commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+We have team-level design team goals to help us align and prioritize our
+focus as a team over the following 3-6 months.
+
+Our team-level goals cascade from the overall company [product vision and
```
I think some clarification on the connection btw the company product vision/strategy and the (design team) themes would be helpful here! At first, I thought the themes were equivalent to the company product strategy, but it seems they're themes derived from the product strategy but more design team specific.

jjinnii commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
+   - Key Result: Eliminate all the UI/UX-related must-haves from the scale
+     audit.
+   - Key Result: Design as a function contributes to closing a deal for a
```
How might we measure this? It could be closing out an issue(s) that is an improvement/feature request that is directly requested by a 2XL customer, but this would be reactive and requires that a customer preemptively requests something that is then tagged as needing design.

quinnkeast replied:
This is intended to be directly tied to the strategic work underway. Assuming we have a specific list of issues (@AlicjaSuska), then measuring it would be tied to addressing these issues.

I'm thinking that as an objective/key result, it should more specifically align it to the strategic effort?

ryphil replied:
One additional fly-by note: Yes, @AlicjaSuska's work is absolutely critical path for strategic. In addition,  every team across the org is working to increase the strategic readiness of their features and design support will be necessary alongside each of those teams as well. For example, BC team will certainly need thoughtful design as they work on the key issues outlined in their [strat readiness doc](https://docs.google.com/document/d/1vXAnhui-AGBk2aar4pXt-uTuTIQfZU5LFFtrMAU2e4s/edit#bookmark=id.9gi05kl6on12) that fall outside of Alicja's central work.

AlicjaSuska replied:
Maybe we could include this in the first KR:
Key Result: Eliminate all the UI/UX-related must-haves from the scale audit.
Key result: Provide design support for all the key issues outlined in the strategic readiness doc for respective teams.

rrhyne replied:
I altered this with a slightly edited version of Alicja's comment and will remove the last KR as I think it's a bridge too far and would be hard to measure. If anyone disagrees, feel free to bring it back.

jjinnii commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+# Design team goals
+
+We have team-level design team goals to help us align and prioritize our
+focus as a team over the following 3-6 months.
```
3-6 months is a pretty varying range in timespan. Is there a reason for this specific timespan? Did we want to align to quarters?

rrhyne replied:
Agreed. These goals were meant to align to the end of Q4.

jjinnii commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+     related to the function of engineering.
+2. **Objective**: Level up storytelling skills.
+   - Key Result: Each member of the design team shares a #progress announcement.
+   - Key Result: Company-wide engagement in design review increases.
```
I feel as if this is more an Objective that the team should aim to accomplish, and one of the KRs could be that design team members share to #progress. Another potential KR (pulling from a design offsite) is using some form of media (video snippets, user interview quotes, etc.) in a #progress post or team discussion.

rrhyne replied:
I tried to capture this by adding: "that exhibit strong storytelling skills."

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+strategy](../../index.md#product-vision-and-strategy), and each member of the design team
+uses these to define their own individual goals.
+
+## (Draft) Design team goals for the next 3-6 months
```
```suggestion
## (Draft) Design team goals for Q4
```

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+#### Theme: Win the top 100 organizations
+
+1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
+   - Key Result: Eliminate all the UI/UX-related must-haves from the scale
```
```suggestion
   - Key Result: Eliminate all the UI/UX-related must-haves from the strategic scale
```

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+2. **Objective**: Get closer to our users.
+   - Key Result: Each member of the design team participates in at least 1
+     sales, AE, or CE session per week.
+   - Key Result: Each member of the design team has at least `${x}` hours of
```
We need to determine what ${x} is. I suggest we use the goal of 1 hour per week (about 2 sessions), and should compensate for the holidays. 

We also need to determine what 'user time' means. I suggest:

> Generative or evaluative research sessions with both internal and external users.

quinnkeast replied:
I also believe at least 1 hour per week is reasonable—perhaps our intent is "per _typical_ week" (which can be defined with an asterisk)?

For "user time," I'd suggest starting with both internal and external users, but in the following quarter, evolving to prefer external users.

How do we consider things like async usertesting.com sessions? Are they part of this time?

rrhyne replied:
I agree on moving primarily toward external. I thnk that user testing session reviews should count toward this time. That said, we'll want to make sure this isn't the only type of testing conducted.

jjinnii replied:
Does time spent synthesizing and tagging this information in Dovetail also part of this time? I think it's important to mention/acknowledge that it takes just as much time to pull out the insights from user interviews into Dovetail properly (I'm already guilty of being behind on this myself 😅 ). I'm not necessarily advocating for fewer interviews, but if we mean 1 hour a week _conducting_ user interviews, realistically that's another half hour to another hour tagging and pulling out insights in Dovetail.

quinnkeast replied:
No, I think this is a measure of time spend with users. I think synthesizing is something we'll get more efficient at as we go.

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+   - Key Result: Each member of the design team participates in at least 1
+     sales, AE, or CE session per week.
+   - Key Result: Each member of the design team has at least `${x}` hours of
+     user time per `${cycle}`.
```
```suggestion
     user time per quarter.
```

Aligning this to the intended cadence.

Commit:
Update content/departments/engineering/design/goals/index.md

Co-authored-by: Rob Rhyne <rob@sourcegraph.com>

Commit:
Update content/departments/engineering/design/goals/index.md

Co-authored-by: Rob Rhyne <rob@sourcegraph.com>

Commit:
Update content/departments/engineering/design/goals/index.md

Co-authored-by: Rob Rhyne <rob@sourcegraph.com>

Commit:
Merge branch 'main' into qk/design-team-goals

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+# Design team goals
+
+We have team-level design team goals to help us align and prioritize our
+focus as a team over the following 3-6 months.
```
```suggestion
focus as a team for the remainder of Q4.
```

Commit:
change to q4

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+2. **Objective**: Get closer to our users.
+   - Key Result: Each member of the design team participates in at least 1
+     sales, AE, or CE session per week.
+   - Key Result: Each member of the design team has at least `${x}` hours of
```
```suggestion
   - Key Result: Each member of the design team has at least 14 hours of
```

Commit:
Update content/departments/engineering/design/goals/index.md

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,67 @@
+2. **Objective**: Get closer to our users.
+   - Key Result: Each member of the design team participates in at least 1
+     sales, AE, or CE session per week.
+   - Key Result: Each member of the design team has at least 14 hours of
```
Note, I set the time goal to 10 hours of user-facing time counting 7 weeks until Jan 31 (minus 2 weeks due to Thanksgiving and Christmas), 45 minutes per session, and 2 sessions per week.

Commit:
remove individual OKR

Remove individual goal OKR and make reference to designers setting their own goals.

rrhyne committed:
squash! Prettier

rrhyne commented:
@quinnkeast  et all, I removed the OKR that related to individual goals and made reference to designers setting their own goals:

https://sourcegraph.com/github.com/sourcegraph/handbook/-/commit/4e5e4037ac3c10970bf70665351bb2f970fdf6ec?visible=1

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,61 @@
+#### Theme: Win the top 100 organizations
+
+1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
+   - Key Result: Eliminate all the UI/UX-related must-haves from the strategic scale
```
```suggestion
   - Key Result: Provide design support for all the key issues determined to be in scope by the strategic readiness effort.
```

Commit:
Update content/departments/engineering/design/goals/index.md

Commit:
Remove 2XL deal OKR

Commit:
Improve progress KR

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,59 @@
+#### Theme: Levelling up the design team
+
+1. **Objective**: Level up storytelling skills.
+   - Key Result: Each member of the design team shares two #progress announcements that exhibt strong storytelling skills.
```
```suggestion
   - Key Result: Each member of the design team shares two #progress announcements that exhibit strong storytelling skills.
```

Commit:
Update content/departments/engineering/design/goals/index.md

Commit:
Slight rewording of intro

Commit:
remove draft

Commit:
Remove global reference to quarter, clarify fiscal year

Commit:
Move brand objective to urgent-important

Commit:
consistency with other themes

Commit:
Properly next brand efforts under new objective

rrhyne committed:
squash! Prettier

rrhyne commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,57 @@
+2. **Objective**: Get closer to our users.
+   - Key Result: Each member of the design team participates in at least 1
+     sales, AE, or CE session per week.
+   - Key Result: Each member of the design team has at least 14 hours of
```
```suggestion
   - Key Result: Each member of the design team has at least 10 hours of
```

Commit:
Update content/departments/engineering/design/goals/index.md

Commit:
Merge branch 'main' into qk/design-team-goals

quinnkeast commented on content/departments/engineering/design/goals/index.md:
```diff
@@ -0,0 +1,57 @@
+
+1. **Objective**: Remove blocks to adoption related to UI/UX for 2XL customers.
+   - Key Result: Provide design support for all the key issues determined to be in scope by the strategic readiness effort.
+     audit.
```
```suggestion
```

Commit:
Update content/departments/engineering/design/goals/index.md

Commit:
Merge branch 'main' into qk/design-team-goals
